netlify.toml

```diff
@@ -1,17 +1,16 @@
 [build]
-  command = "remix build"
-  publish = "public"
+command = "npm run build"
+publish = "build/client"
```

Comment:
I'm not sure if this is correct? In the pipeline log (I think it is publicly accessible? => https://app.netlify.com/sites/keen-liskov-77de74/deploys/66eb34069a3c1f0008085b6b ), there's only the client part mentioned in the deployment step. But we definitely also need the server part. That should be the reason why you cannot access the page.

Reply:
Yeah, but I had the same thought as well and checked it that to be sure :D